### DISCUSSION TITLE
Can publish to scoop so I can install using scoop?

### DISCUSSION THREAD

**Discussion by gabrielkryss** [lin-ycv/EverythingPowerToys#100] [General]
I usually use scoop to install my stuff I'd hate to install choco, just to install one plugin hence I wonder if it's possible to have it on scoop?

**Comment by lin-ycv** [comment]
Tried this before, but scoop would not accept the submission, don't know why. 
The PR is still open https://github.com/ScoopInstaller/Extras/pull/12912

UPDATE: Scoop submission has been accepted.
